client/reader/controller.js

```diff
@@ -349,3 +352,7 @@ export const {
 	readA8C,
 	readFollowingP2,
 } = exported;
+
+export const unreadFollowing = ( context, next ) => exported.following( context, next, false );
+export const unreadA8C = ( context, next ) => exported.readA8C( context, next, false );
+export const unreadFollowingP2 = ( context, next ) => exported.following( context, next, false );
```

Comment:
should this use `exported.readFollowingP2()`? right now `unreadFollowing` and `unreadFollowingP2` do the same thing

Reply:
Does this still apply @blackjackkent ?